### PR TITLE
Add meeting minutes for 2017-11-30

### PR DESCRIPTION
**Pull request by bnb**
Creates the file in the repo for last week's meeting. Original source: https://docs.google.com/document/d/1V9hFrZ4q8nIZj8yMveipZ5PKqqwZHZGNV4uPR7QnJ1o/edit?usp=sharing

### CODE REVIEW

File: meetings/2017-11-30.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+Tierney Cyren (@bnb - CommComm)
+Tracy Hinds (@hackygolucky - CommComm)
+Olivia Hugger (@pup - CommComm)
+Ben Michel (Observer ?)
+Dan Shaw (Obverser, End user feedback initiative)
+Jem bezooyen (Observer)
+Manil (Observer)
+Nikhil Komawar (Observer)
+Pooya Paridel (Observer)
+Rafel Ackermann(@refack CommComm)
```
Comment (pup) [inline meetings/2017-11-30.md]: missing space here

File: meetings/2017-11-30.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+Ahmad Bamieh (Observer)
+Benjamin Zaslavsky (@tiriel - Observer)
+
+
+
+## Notes
+
+Extracted from cc-agenda labelled issues and pull requests from the nodejs org prior to the meeting.
```
Comment (pup) [inline meetings/2017-11-30.md]: all of this needs to be formatted as bullet points, markdown ignores single line breaks

**Comment by bnb** [comment]
@pup Updated, fixed a lot of styling. PTAL 🙏

**Comment by pup** [comment]
oh yeah, should we put `closes [meeting issue]` when we submit meeting notes? sometimes we forget to close the meeting issue even after merging the notes, but this would do it automatically

**Comment by bnb** [comment]
@pup Yep! That's a great idea - I didn't actually know what the syntax was, but now I do. We should _probably_ log that somewhere. Does it need to be in the actual commit message?

Going to merge this and close https://github.com/nodejs/community-committee/issues/179

**Comment by pup** [comment]
@bnb iirc github added a feature where if you just type "closing [issue]", it'll auto-close it

**Pull request merged**